color-picker-project/src/components/colorpicker/ColorPicker.css

```diff
@@ -0,0 +1,45 @@
+.color-picker-div {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.color-picker-container {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    background-color: lightyellow;
+    border: solid;
+    border-width: 1px;
+}
+
+.color-picker-wheel {
+    margin-top: 2%;
+}
+
+.color-picker-color {
+    margin-bottom: 5%;
+}
+
+.color-picker-material {
+    width: 200px;
+    margin-top: 1.5%;
+}
+
+.color-picker-title {
+    margin-top: 5%;
+    font-family: 'Times New Roman', Times, serif;
```

Comment:
Since the rest of the app is using Poppins as the font we probably shouldn't have this title using Times New Roman.